valhalla/app/views/valhalla/base/_form_progress.html.erb

```diff
@@ -16,16 +16,19 @@
       <%= render 'form_visibility_component', f: f %>
     </div>
   </div>
-  <div class="panel-footer text-center">
-    <br>
-    <%# send user back to object from edit, back to home from new %>
```

Comment:
this merged over #432

Reply:
it happens! I just opened a https://github.com/pulibrary/figgy/pull/437 to get the fix back in.